dev-java/openjdk/openjdk-11.0.14_p9-r1.ebuild

```
@@ -36,11 +37,14 @@ SRC_URI="
 	!system-bootstrap? (
 		$(bootstrap_uri ppc64 ${PPC64_XPAK} big-endian)
 		$(bootstrap_uri x86 ${X86_XPAK})
+		riscv? ( https://github.com/Rabenda/gentoo-riscv-bootstarp-package/releases/download/bootstrap-openjdk-11/openjdk-bootstrap-11.0.15-riscv.tar.xz )
```

I have uploaded this tarball to my devspace, so I think this will work now 😄 

```suggestion
				$(bootstrap_uri riscv ${RISCV_XPAK})
```

Also note that you don't need to rebuild the xpak every version bump, and it should work well for all `openjdk:11` future versions, so I don't think we will need to recreate xpak every bump - makes it easier to maintain 😄

---

Then I modify it.

---

Sorry, I don't have any riscv boxed to compile it on it. Shortly, the process is to install all deps beforehand for this package (you can use your current bootstrap tarball for that)

And then build using: `XPAK_BOOTSTRAP="bundled" USE="jbootstrap" emerge -1a --buildpkg=y =dev-java/openjdk:11::gentoo`
This will build once using your tarball, and than build second time using first build. By setting the `XPAK_BOOTSTRAP`, it will bundle most things so it is safer ABI wise.

After that, you can find the prepare tarball script [here](https://dev.gentoo.org/~arthurzam/prepare-openjdk-bootstrap.sh). See help there on how to use. Maybe you will need to change `tar -x --zstd` to correct compression algorithm on you machine.

And then if you created a new tarball, send me the latest link and I will reupload to my devspace.